src/pykeen/nn/modules.py

```diff
@@ -1643,16 +1653,40 @@ def _prepare_hrt_for_functional(
         return dict(h=h, r=r, c_r=c_r, t=t)
 
 
+@parse_docdata
 class BoxEInteraction(
     NormBasedInteraction[
         Tuple[FloatTensor, FloatTensor],
         Tuple[FloatTensor, FloatTensor, FloatTensor, FloatTensor, FloatTensor, FloatTensor],
         Tuple[FloatTensor, FloatTensor],
     ]
 ):
-    """An implementation of the BoxE interaction from [abboud2020]_."""
+    """
+    The BoxE interaction.
```

Comment:
it's good to keep the `[abboud2020]_` reference since the part in markdown doesn't get shown in sphinx